broker/lib/fabrik/CfPlatformManager.js

```diff
@@ -114,11 +114,22 @@ class CfPlatformManager extends BasePlatformManager {
       );
   }
 
+  /*
+  {
+    protocol: 'tcp',
+    ips: ['10.11.20.248', '10.11.20.255'],
+    applicationAccessPorts: ['8080']
+  }
+  */
   buildSecurityGroupRules(options) {
+    let portRule = '1024-65535';
+    if (options.applicationAccessPorts && _.size(options.applicationAccessPorts) > 0) {
+      portRule = _.size(options.applicationAccessPorts) === 1 ? `${_.first(options.applicationAccessPorts)}` : `${_.join(options.applicationAccessPorts,',')}`;
```

Comment:
`_.join(options.applicationAccessPorts,',')` is enough. Even if array size is 1 produces same result.

Reply:
Okay. Will update it as well. Thanks